contrib/podmanimage/upstream/Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN useradd podman; yum -y update; yum -y reinstall shadow-utils; yum -y install
      cd $GOPATH/src/github.com/containernetworking/plugins; \
      ./build_linux.sh; \
      mkdir -p /usr/libexec/cni; \
-     \cp -fR bin/* /usr/libexec/cni; \
+     /usr/bin/cp -fR bin/* /usr/libexec/cni; \
```

Comment:
Personally I don't think this is necessary, and I don't believe that it's at all related to the rest of this PR.  Please remove this.  If you really feel it's necessary, please change it in a separate PR so we can track the change in Git history.

Reply:
thx